x11-misc/idesk/idesk-1.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Utility to place icons on the root window"
+HOMEPAGE="https://github.com/antonialoytorrens/idesk/"
+SRC_URI="https://github.com/antonialoytorrens/idesk/releases/download/v1/idesk-v1.tar.gz -> ${P}.tar.gz"
```

Comment:
You need either to use ver_cut here or rename ebuild to idesk-1.ebuild and use here ${PV} as version substitute.

Reply:
What is ver_cut? I have not understand that thing yet

Reply:
I have also seen here: https://devmanual.gentoo.org/ebuild-writing/variables/index.html

Reply:
Fixed